src/style.css

```diff
@@ -0,0 +1,21 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.block-size {
+  height: 100px;
+  width: 100%;
```

Comment:
Block elements has 100% of the width of the parent container. Set width of 100% is redundant